ext/ffi/00_ffi.js

```diff
@@ -13,8 +13,29 @@
       this.#rid = core.opSync("op_ffi_load", { path, symbols });
 
       for (const symbol in symbols) {
-        this.symbols[symbol] = (...parameters) =>
-          core.opSync("op_ffi_call", { rid: this.#rid, symbol, parameters });
+        this.symbols[symbol] = (...args) => {
+          const parameters = [];
+          const buffers = [];
+
+          for (const arg of args) {
+            if (
+              arg && arg.buffer instanceof ArrayBuffer &&
```

Comment:
Couldn't this be simplified to `arg?.buffer instanceof ArrayBuffer && ...`?

Reply:
You're right 👍